diff --git a/node_modules/@bazel/typescript/internal/ts_project.bzl b/node_modules/@bazel/typescript/internal/ts_project.bzl
index 2fa84ac..5108b64 100755
--- a/node_modules/@bazel/typescript/internal/ts_project.bzl
+++ b/node_modules/@bazel/typescript/internal/ts_project.bzl
@@ -338,6 +336,9 @@ def _is_ts_src(src, allow_js):
 def _is_json_src(src, resolve_json_module):
     return resolve_json_module and src.endswith(".json")
 
+def _without_bin(src):
+    return src.replace("_bin_copy", "")
+
 def _replace_ext(f, ext_map):
     cur_ext = f[f.rindex("."):]
     new_ext = ext_map.get(cur_ext)
@@ -352,6 +353,7 @@ def _out_paths(srcs, outdir, rootdir, allow_js, ext_map):
     rootdir_replace_pattern = rootdir + "/" if rootdir else ""
     outs = []
     for f in srcs:
+        f = _without_bin(f)
         if _is_ts_src(f, allow_js):
             out = _join(outdir, f[:f.rindex(".")].replace(rootdir_replace_pattern, "") + _replace_ext(f, ext_map))
 
