diff --git a/node_modules/@bazel/typescript/internal/ts_project.bzl b/node_modules/@bazel/typescript/internal/ts_project.bzl
index a6342f1..bd48511 100755
--- a/node_modules/@bazel/typescript/internal/ts_project.bzl
+++ b/node_modules/@bazel/typescript/internal/ts_project.bzl
@@ -336,11 +336,15 @@ def _replace_ext(f, ext_map):
 
 def _out_paths(srcs, outdir, rootdir, allow_js, ext_map):
     rootdir_replace_pattern = rootdir + "/" if rootdir else ""
-    return [
-        _join(outdir, f[:f.rindex(".")].replace(rootdir_replace_pattern, "") + _replace_ext(f, ext_map))
-        for f in srcs
-        if _is_ts_src(f, allow_js)
-    ]
+    out = []
+    for f in srcs:
+        f = _without_bin(f)
+        if _is_ts_src(f, allow_js):
+            out.append(_join(outdir, f[:f.rindex(".")].replace(rootdir_replace_pattern, "") + _replace_ext(f, ext_map)))
+    return out
+
+def _without_bin(src):
+    return src.replace("_bin_copy", "")
 
 def ts_project_macro(
         name = "tsconfig",
